app/views/europe_at_work/_form.html.erb

```diff
@@ -0,0 +1,34 @@
+<% with_options scope: 'contribute.campaigns.migration.form' do |i18n| %>
```

Comment:
As this just duplicates the migration form, for now just:
```
<%= render partial: '/migration/form' %>
```

... and remove app/views/europe_at_work/field_sets, and in the migration form partial, add `/migration/` to each of the field set partials render commands.